lib/serverengine.rb

```diff
@@ -47,6 +47,8 @@ module ServerEngine
     require File.join(here, v)
   }
 
+  $platformwin = /mswin|mingw/ === RUBY_PLATFORM
```

Comment:
using global variable is not good idea because its name is not in a namespace of ServerEngine.
Why don't you use this way?:

```
module ServerEngine
  def self.windows?
    IS_WINDOWS
  end

  IS_WINDOWS = /mswin|mingw/ === RUBY_PLATFORM
  private_constant :IS_WINDOWS
end
```